### ISSUE TITLE
EDSC-960: Limitations on order sizes and expected behavior need to be more clear

### ISSUE THREAD

**Issue by earthdata-tasktop** [nasa/edsc-search-test-config-3#54]

**Comment by earthdata-tasktop** [comment]
(Comment from Bo Sun):

A popup dialog/modal will be shown if user orders more than 2000 granules.

Use these collections in UAT:

* C1000-LPDAAC_TS2
* C1052-LPDAAC_TS1
  ##### Steps to Verify
* One collection, one access method:
* search for C1000-LPDAAC_TS2
* click "Retrieve collection data"
* on /data/configure page, select **AST_L1A** access option
* click continue, you should see a popup

* Refine your search: will send you back to granule list page
* Change access method will dismiss the popup
* Continue will continue to the next step
* Repeat step 1 and 2, but choose **AST_L1A.3 ESI Service** this time, click continue. You won't see the popup since it is not an "order" type.
* Two collections, two different access methods
* Use both collections above

* Search each collection and click '+' to add it to a project, then click 'View project'
* Follow similar steps in "One collection, one access method"
* One collection, multiple access methods
* Follow similar steps in "One collection, one access method", but on /data/configure page, click "Add access method" button to add another one.
* As long as there is one access method which type is 'order', when "continue" is clicked, the popup will be shown.

**Comment by earthdata-tasktop** [comment]
(Comment from Kathleen Carr):

Verified that, in addition to the red message indicating that orders > 2000 granules are subject to limitations, there is now a pop-up dialog which gives the user that information when they attempt to place an order for > 2000 granules.

I verified that the three buttons on the pop-up (Refine your search, change access method, and Continue) work properly, and that the pop-up can be closed by clicking on the x.

I also tested with 

* One collection, one access method
* Two collections in a project (each > 2000), one access method per collection 
* Two collections in a project (each < 2000 granules, but total > 2000 granules) - the orders are placed by collection, not by project, so the pop-up did not appear
* One collection, multiple access methods 

Behavior was as specified in the Verification instructions.
